crossbar/_logging.py

```diff
@@ -353,7 +347,7 @@ def color_json(json_str):
     Given an already formatted JSON string, return a colored variant which will
     produce colored output on terminals.
     """
-    assert(type(json_str) == six.text_type)
+    assert(type(json_str) == str)
     return highlight(json_str, lexers.JsonLexer(), formatters.TerminalFormatter())
```

Comment:
Not this PR's fault, but asserts like this should really be `assert isinstance(json_str, str)` or so